views/agency-requests/view_participant.php

```diff
@@ -13,7 +13,7 @@
  * @version 0.1.6
  * @since 0.1
  */
-
+include('header.php');
```

Comment:
I would suggest putting this statement right before the HTML portion of your page starts (line 43ish)